cogs/Misc.py

```diff
@@ -187,9 +206,68 @@ async def edit(text: str) -> None:
                 file=file,
                 ephemeral=True,
             )
-            await edit(f"{LOADING_EMOJI} Dumped {i + 1}/{len(tables)} table(s).")
+            await edit(f"{LOADING_EMOJI} Presented {i + 1}/{len(tables)} table(s).")
+
+        await edit(f"{COMPLETED_EMOJI} Presented {len(tables)} table(s).")
 
-        await edit(f"{COMPLETED_EMOJI} Dumped {len(tables)} table(s).")
+    @VIEW_LOGS_PERMISSIONS
+    @commands.slash_command(
+        name="view-logs",
+        description="View the most recent logs generated by the bot.",
+        guild_ids=None if VIEW_DB_GUILD is None else [VIEW_DB_GUILD],
+        guild_only=VIEW_DB_GUILD is not None,
+    )
+    @option(
+        "file",
+        str,
+        parameter_name="filename",
+        description="The log file you want to view.",
+        required=True,
+        default="log.txt",
+        choices=[
+            OptionChoice(f"{filename} - {description}", filename)
+            for filename, description in LOG_FILES.items()
+        ],
+    )
+    @option(
+        "lines",
+        int,
+        parameter_name="line_count",
```

Comment:
`Line count` feels ambiguous, lines from the start? From the end? I suggest changing this argument to `tail` , like the Linux command.